sig-ui/charter.md

```diff
@@ -0,0 +1,44 @@
+# SIG UI Charter
+
+This charter is a WIP.
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+SIG-UI covers all things user interface related within the Kubernetes project. Efforts are centered around the Kubernetes Dashboard: a general purpose, web-based UI for Kubernetes clusters. It allows users to manage applications running in the cluster and troubleshoot them, as well as manage the cluster itself.
+
+### In scope
+
+  Link to SIG section in [sigs.yaml]
+
+#### Code, Binaries and Services
+
+- Kubernetes Dashboard
```

Comment:
Just kubernetes/dashboard or also the addon in k/k's cluster/addons/dashboard?

Reply:
Regarding add-ons:
- https://github.com/kubernetes/minikube/tree/master/deploy/addons
- https://github.com/kubernetes/kops/tree/master/addons/kubernetes-dashboard
- https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/dashboard

Reply:
Yeah I guess I would suggest listing that you also own the addons.  Your current process sounds like a fine way of doing it, and iterating to the former process sounds like a good next step.